### PR TITLE
Draft count 

### PR DESCRIPTION
**Pull request by seethalelias-aot**
Added draft count to the application endpoint.
Postman Response:
image

DraftDB:
draft

Application DB:
application

### CODE REVIEW

File: forms-flow-api/src/formsflow_api/models/application.py [hunk 1]
```diff
@@ -129,6 +129,9 @@ def find_all(
         """Fetch all application."""
         query = cls.filter_conditions(**filters)
         query = FormProcessMapper.tenant_authorization(query=query)
+        draft_count = query.filter(
+            Application.application_status == DRAFT_APPLICATION_STATUS
```
Comment (sreehari-aot) [inline forms-flow-api/src/formsflow_api/models/application.py:133]: need to fetch the count for active drafts only, add that too

File: forms-flow-api/src/formsflow_api/models/application.py [hunk 2]
```diff
@@ -194,8 +197,11 @@ def find_all_by_user(  # pylint: disable=too-many-arguments
         """Fetch applications list based on searching parameters for Non-reviewer."""
         query = cls.filter_conditions(**filters)
         query = FormProcessMapper.tenant_authorization(query=query)
-        query = cls.filter_draft_applications(query=query)
         query = query.filter(Application.created_by == user_id)
+        draft_count = query.filter(
+            Application.application_status == DRAFT_APPLICATION_STATUS
```
Comment (sreehari-aot) [inline forms-flow-api/src/formsflow_api/models/application.py:202]: same

File: forms-flow-api/src/formsflow_api/models/application.py [hunk 3]
```diff
@@ -296,6 +302,10 @@ def find_applications_by_process_key(  # pylint: disable=too-many-arguments
         query = cls.filter_conditions(**filters)
         query = FormProcessMapper.tenant_authorization(query=query)
         query = query.filter(FormProcessMapper.process_key.in_(process_key))
+        draft_count = query.filter(
+            Application.application_status == DRAFT_APPLICATION_STATUS
```
Comment (sreehari-aot) [inline forms-flow-api/src/formsflow_api/models/application.py:306]: same

**Review by sreehari-aot — COMMENTED** [review]
add / update the test cases as well

**Review by abhilash-aot — APPROVED** [review]
Any changes need in frontend @sreehari-aot

**Comment by sreehari-aot** [comment]
can u attach a response screenshot

**Review by AbijithS-aot — APPROVED** [review]

**Review by sreehari-aot — APPROVED** [review]

**Review by auslin-aot — APPROVED** [review]

**Review by sreehari-aot — CHANGES_REQUESTED** [review]
correction needed

**Review by sreehari-aot — APPROVED** [review]

**Pull request merged**